whedon_api.rb

```diff
@@ -59,6 +59,10 @@ def testing?
     ENV['RACK_ENV'] == "test"
   end
 
+  def serialized_config
```

Comment:
We need to serialize the config `OpenStruct` object to a hash before creating the Sidekiq job.